Documentation/Contributing/BreakingChanges.md

```diff
@@ -1,2 +1,129 @@
 # Breaking Changes
 
+Consumers of the MRTK depend on having a stable release-to-release API surface, so that
+they can take updates to the MRTK without having large breaking changes each time.
+
+This page describes our current policy regarding breaking changes in the MRTK, along with some
+longer term goals around how we can better manage the tradeoff between keeping breaking changes
+low and being able to make the right long term technical changes to the code.
+
+## What is a breaking change?
+
+A change is a breaking change if it satisfies any of the conditions in the [List A](#list-a) AND satisfies all of
+the conditions in [list B](#list-b)
+
+### List A
+
+- The addition, removal, or update of any member or function of any interface (or removal/rename
+  of the entire interface).
+- The removal, update (changing type/definition, making private or internal) of any protected or public
+  member or function of class. (or removal/rename of the entire class).
+- The change in the order of events fired by a class.
+- The rename of any private SerializedField (without a corresponding FormerlySerializedAs tag) or public
+  property on a ScriptableObject (especially changes to profiles).
+- Changing the type of a field on a ScriptableObject (especially changes to profiles).
+- Updates to the namespace or asmdefs of any class or interface.
+- Removal of any prefab or removal of a script on the top level object of a prefab.
+
+### List B
+
+- The asset in question is in the Foundation package (i.e. it's in one of the following folders):
+
+  - MixedRealityToolkit/
+  - MixedRealityToolkit.Providers/
+  - MixedRealityToolkit.Services/ 
+  - MixedRealityToolkit.SDK/
+
+- The asset in question does not belong to the Experimental namespace.
+
+> Any asset that sits in the Examples package (i.e. part of the MixedRealityToolkit.Examples/ folder)
+> is subject to change at any time, as assets there are designed to be copied and viewed by consumers
+> as 'reference implementations' but are not part of the core set of APIs and assets.
+> Assets in the experimental namespace (or more generally, features labelled as experimental) are ones
+> that get published before all due diligence has been done (i.e. tests, UX iteration, documentation) and
+> is published early to get feedback sooner.  However, because they don't have tests and documentation, and
+> because we likely haven't nailed down all of the interactions and designs, we publish them in a state
+> where the public should assume that they can and will change (i.e. be modified, completely removed, etc).
+> 
+> See [Experimental Features](../ExperimentalFeatures.md) for more information.
+
+Note that surface area for breaking changes is very large, it's important to note that having an absolute
```

Comment:
```suggestion
As the surface area for breaking changes is very large, it's important to note that having an absolute
```
(there's two times note in that sentence, so i would get rid of the first one)

Reply:
yay